coalahtml/tree.py

```diff
@@ -0,0 +1,81 @@
+from collections import OrderedDict
+from queue import Queue
+import os
+
+
+class Node:
+    """
+    Node class that defines an individual node.
+    """
+
+    def __init__(self, path):
+        self.child = {}
+        self.parent = None
+        self.path = path
+        self.abspath = ""
+        self._id = id(self)
```

Comment:
It seems you only use this for comparison, so you can use `==`. If I remember correctly it compares hashes by default, so you're safe with it.